api/src/lib/aws.ts

```diff
@@ -61,7 +62,7 @@ function getS3Client() {
 export function uploadWorkbook(
   upload: CreateUploadInput,
   uploadId: number,
-  body: any
+  body: StreamingBlobPayloadInputTypes
```

Comment:
Using types from [`@smithy/types`](https://www.npmjs.com/package/@smithy/types) as a more descriptive alternative. 

This way, type-checkers can determine whether `body` values are compatible with the S3 PutObject operation.